packages/venia-concept/src/components/PurchaseDetailsPage/OrderItem/ButtonGroup/__tests__/buttonGroup.spec.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ButtonGroup from '../buttonGroup';
+import Button from 'src/components/Button';
+
+configure({ adapter: new Adapter() });
+
+const classes = {
+    buttonGroupItem: 'buttonGroupItem'
+};
+
+const onClickHandler = jest.fn();
+
+const buttonGroupItemsMock = [
+    {
+        onClickHandler: onClickHandler,
+        iconName: 'icon',
+        textNode: 'text'
+    },
+    {
+        onClickHandler: onClickHandler,
+        iconName: 'icon',
+        textNode: 'text'
+    },
+    {
+        onClickHandler: onClickHandler,
+        iconName: 'icon',
+        textNode: 'text'
+    },
+    {
+        onClickHandler: onClickHandler,
+        iconName: 'icon',
+        textNode: 'text'
+    }
+];
+
+test('renders correctly with 4 buttons', () => {
+    const wrapper = shallow(
+        <ButtonGroup buttonGroupItems={buttonGroupItemsMock} />
+    ).dive();
+
+    expect(wrapper.find(Button)).toHaveLength(buttonGroupItemsMock.length);
+});
+
+test('renders correctly with 1 button', () => {
```

Comment:
This test cases for 1 and 4 buttons are the same. You can just replace them with one test case for some number of buttons.

```
test('renders all buttonGroupItems', () => { ... });
```